build-plugin/action.yml

```diff
@@ -0,0 +1,148 @@
+name: "Grafana Build Plugin"
```

Comment:
this is fine if Grafana Build Plugin is the official name of the tool/app. Otherwise, avoid Title Case.

Reply:
Indeed. this is the title It'll appear in the CI when it runs